src/Compilers/CSharp/Portable/Symbols/NamedTypeSymbol.cs

```diff
@@ -751,6 +739,11 @@ private bool EqualsComplicatedCases(NamedTypeSymbol other, TypeCompareKind compa
                 }
             }
 
+            if (this.IsTupleType && !this.TupleNamesEquals(other, comparison))
```

Comment:
>TupleNamesEquals [](start = 42, length = 16)

Where is `TupleNamesEquals` declared?

Reply:
In `TupleTypeSymbol.cs`

---
In reply to: [355074654](https://github.com/dotnet/roslyn/pull/39370#discussion_r355074654) [](ancestors = 355074654)